website/integrations/services/freshrss/index.md

```diff
@@ -0,0 +1,63 @@
+---
+title: FreshRSS
+---
+
+<span class="badge badge--secondary">Support level: Community</span>
+
+## What is FreshRSS
+
+> FreshRSS is a self-hosted RSS feed aggregator.
+>
+> -- https://github.com/FreshRSS/FreshRSS
+
+## Preparation
+
+The following placeholders will be used:
+
+-   `service.company` is the FQDN of the FreshRSS install.
+-   `port` is the port on which the FreshRSS install is running (usually 443)
+-   `authentik.company` is the FQDN of the authentik install.
+
+## authentik Configuration
+
+In Authentik, create an _OAuth2/OpenID Provider_ under _Applications > Providers_.
+
+** Protocol Settings **
+_Client Type_ : _Confidential_
+
+:::note
+Take note of the `Client ID` and `Client Secret`, you'll need them later.
+:::
+
+_Redirect URIs/Origins_ :
+
+-   `https://service.company/i/oidc/`
+-   `https://service.company:port/i/oidc`
+
+_Signing Key_ : Any of your signing keys
+
+Then click _Finish_ to create your provider.
+
+Then create an _Application_, note its slug, and assign it to the provider you've just created.
+
+## FreshRSS Configuration
+
+:::info
+This integration only works with the Docker or Kubernetes install of FreshRSS, using [FreshRSS docker image](https://hub.docker.com/r/freshrss/freshrss/), on x86_64 systems and with the Debian version of the image.
+:::
+
+Add those environment variables to your _Docker_ image :
+
+-   `OIDC_ENABLED` : `1`
+-   `OIDC_PROVIDER_METADATA_URL` : `https://authentik.company/application/o/slug/.well-known/openid-configuration` replacing slug with the slug of your created application
+-   `OIDC_CLIENT_ID` : the client ID of your provider
+-   `OIDC_CLIENT_SECRET` : the client secret of your provider
+-   `OIDC_X_FORWARDED_HEADERS` : `X-Forwarded-Port X-Forwarded-Proto X-Forwarded-Host`
```

Comment:
This seems odd. That should be a reverse proxy configuration.

Reply:
> This seems odd. That should be a reverse proxy configuration.

What do you mean? About the `OIDC_X_FORWARDED_HEADERS`? Apache OIDC extension needs them behind the reverse proxy

Reply:
I see. We're linking to the FreshRSS documentation which has all the relevant details, so fine by me.